optuna/samplers/_cmaes.py

```diff
@@ -144,6 +153,16 @@ def objective(trial):
                 used. Please see `the benchmark result
                 <https://github.com/optuna/optuna/pull/1229>`_ for the details.
 
+        use_separable_cma:
+            If this is :obj:`True`, the covariance matrix is constrained to be diagonal.
+            Due to reduce the model complexity, the learning rate for the covariance matrix
+            is increased. Consequently, this algorithm outperforms CMA-ES on separable functions.
+
+            .. note::
+                Added in v2.3.0 as an experimental feature. The interface may change in newer
+                versions without prior notice. See
+                https://github.com/optuna/optuna/releases/tag/v2.3.0.
```

Comment:
Could you update the version number?
```suggestion
                Added in v2.6.0 as an experimental feature. The interface may change in newer
                versions without prior notice. See
                https://github.com/optuna/optuna/releases/tag/v2.6.0.
```